source/adapters/hip/enqueue.cpp

```diff
@@ -92,30 +92,46 @@ ur_result_t setHipMemAdvise(const void *DevPtr, const size_t Size,
   if (URAdviceFlags &
       (UR_USM_ADVICE_FLAG_SET_NON_ATOMIC_MOSTLY |
        UR_USM_ADVICE_FLAG_CLEAR_NON_ATOMIC_MOSTLY |
-       UR_USM_ADVICE_FLAG_BIAS_CACHED | UR_USM_ADVICE_FLAG_BIAS_UNCACHED)) {
+       UR_USM_ADVICE_FLAG_BIAS_CACHED | UR_USM_ADVICE_FLAG_BIAS_UNCACHED
+#if !defined(__HIP_PLATFORM_AMD__)
```

Comment:
Is it worth it to add `ifdef` conditions for this? I thought that we don't really support HIP for Nvidia devices.

Reply:
We don't actively support it but until we have completely dropped it officially there are a bunch of other entry points in the hip adapter that that maintain working codepaths for hip-nv platforms. Either way, I do not disagree to removing it and simplifying this function a bit this way.